app/assets/stylesheets/stream_element.scss

```diff
@@ -60,10 +60,45 @@
   }
 }
 
+@keyframes fadein {
+    from {
+        opacity:0;
+    }
+    to {
+        opacity:1;
+    }
+}
+@-moz-keyframes fadein { /* Firefox */
+    from {
+        opacity:0;
+    }
+    to {
+        opacity:1;
+    }
+}
+@-webkit-keyframes fadein { /* Safari and Chrome */
+    from {
+        opacity:0;
+    }
+    to {
+        opacity:1;
+    }
+}
+@-o-keyframes fadein { /* Opera */
+    from {
+        opacity:0;
+    }
+    to {
+        opacity: 1;
+    }
+}
+
 #main_stream .stream-element {
   margin-bottom: 20px;
-  border: 1px solid $border-grey;
-  box-shadow: $card-shadow;
+  animation: fadein 2s;
+-moz-animation: fadein 2s; /* Firefox */
+-webkit-animation: fadein 2s; /* Safari and Chrome */
```

Comment:
- Use `//` comments everywhere
- Line should be indented 2 spaces, but was indented 0 spaces
- Avoid vendor prefixes.